clang/test/Preprocessor/sycl-macro.cpp

```diff
@@ -7,6 +7,13 @@
 // RUN: %clang_cc1 -fno-sycl-id-queries-fit-in-int %s -E -dM | FileCheck \
 // RUN: --check-prefix=CHECK-NO-SYCL_FIT_IN_INT %s
 
+// -fsycl-id-queries-fit-in-int is ON by default
```

Comment:
I don't think the run lines 2-6 are needed now that it is on by default.

Reply:
Removed.